labelbox/schema/data_row_metadata.py

```diff
@@ -147,6 +147,31 @@ def _build_ontology(self):
             str, DataRowMetadataSchema] = self._make_normalized_name_index(
                 self.custom_fields)
 
+    def get_by_name(
+        self, name: str
+    ) -> Union[DataRowMetadataSchema, Dict[str, DataRowMetadataSchema]]:
+        """ Get metadata by name
+
+        >>> mdo.get_by_name(name)
+
+        Args:
+            name (str): Name of metadata schema
+
+        Returns:
+            Metadata schema as `DataRowMetadataSchema` or dict, in case of nested metadata
```

Comment:
- `in case of Enum metadata` is probably more accurate
- It'd be nice to provide an example output of calling get_by_name

Reply:
Thanks! I have changed the text as you suggesting. 
Separately, where do you want me to provide an example? I can not find any instance of Example in the repo. I will update the SDK doc of course after we release this version.